.travis.yml

```diff
@@ -8,15 +8,8 @@ cache:
 
 matrix:
     include:
-        - php: 5.4
-          env: check_cs=true
-        - php: 5.5
-        - php: 5.6
-          env: deps=low coverage=true
-        - php: 5.6
-          env: deps=high coverage=true
-        - php: 7.0
         - php: 7.1
+          env: deps=low check_cs=true coverage=true
```

Comment:
I would prefer to not run the check_cs with the lowest dependencies.

Reply:
Sure I changed it.
For curiosity, any particular reason for not running code style check with lowest dependencies?